### PR TITLE
webui: Show task fails and allow bulk restart on PoRep page

### PR DESCRIPTION
**Pull request by magik6k**
2024-08-25-115303_1096x1661_scrot

### CODE REVIEW

File: web/static/pages/node_info/node-info.mjs [hunk 1]
```diff
@@ -111,7 +111,7 @@ customElements.define('node-info',class NodeInfoElement extends LitElement {
                         <td>${task.ID}</td>
                         <td>${task.Task}</td>
                         <td>${task.Posted}</td>
-                        <td>${task.PoRepSector ? html`<a href="/pages/pipeline_porep/">f0${task.PoRepSectorSP}:${task.PoRepSector}</a>` : ''}</td>
+                        <td>${task.PoRepSector ? html`<a href="/pages/sector/?sp=f0${task.PoRepSectorSP}&id=${task.PoRepSector}">f0${task.PoRepSectorSP}:${task.PoRepSector}</a>` : ''}</td>
```
Comment (snadrus) [inline web/static/pages/node_info/node-info.mjs:114]: Is that f0 compatible with calibrationNet?

File: web/static/pages/pipeline_porep/restart-all-button.mjs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        this.isProcessing = true;
+        try {
+            await RPCCall('PipelinePorepRestartAll', []);
+            console.log('Restart All operation completed successfully');
+        } catch (error) {
+            console.error('Error during Restart All operation:', error);
+        } finally {
+            this.isProcessing = false;
+        }
+    }
```
Comment (snadrus) [inline web/static/pages/pipeline_porep/restart-all-button.mjs:37]: this.requestUpdate();

**Review by snadrus — APPROVED** [review]

File: web/static/pages/node_info/node-info.mjs (hunk 1 above)
Comment (magik6k) [reply web/static/pages/node_info/node-info.mjs:114]: yep, parser doesn't care

File: web/static/pages/pipeline_porep/restart-all-button.mjs (hunk 2 above)
Comment (magik6k) [reply web/static/pages/pipeline_porep/restart-all-button.mjs:37]: Not needed for stuff listed in `static properties`

**Pull request merged**